stage1/init/common/pod.go

```diff
@@ -456,9 +456,8 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 	}
 
 	if flavor == "kvm" {
-		// bind mount all shared volumes from /mnt/volumeName (we don't use mechanism for bind-mounting given by nspawn)
-		err := AppToSystemdMountUnits(common.Stage1RootfsPath(p.Root), appName, p.Manifest.Volumes, ra, UnitsDir)
-		if err != nil {
+		// bind mount all shared volumes (we don't use mechanism for bind-mounting given by nspawn)
+		if err := MountSharedVolumes(common.Stage1RootfsPath(p.Root), appName, p.Manifest.Volumes, ra); err != nil {
```

Comment:
Since `MountSharedVolumes` is not related to systemd-nspawn parameters nor systemd units, I think the function call should not be in `stage1/init/common/pod.go:appToSystemd` but directly in `stage1/init/init.go:stage1()`. It would require to write another loop on p.Manifest.Apps though. What do you think?